lifesaver/cli.py

```diff
@@ -17,6 +17,17 @@ def resolve_class(specifier: str):
     return loaded_class
 
 
+async def _postgres_connect(bot):
```

Comment:
The method should be kept on the bot as that's where the database lives. It should still be called from `lifesaver.cli` though.